doc/topics/proxyminion/index.rst

```diff
@@ -36,6 +36,21 @@ or more minions.
 See :ref:`Proxyminion Beacon <proxy-minion-beacon>` to help
 with easy configuration and management of ``salt-proxy`` processes.
 
+New in Nitrogen
```

Comment:
I think we should either copy or move this to the release notes for the Nitrogen release itself. Otherwise, it's just too easy to miss this upon upgrading.